docs/get-started/sign-up.md

```diff
@@ -27,25 +26,25 @@ The link in your account activation email will expire after 3 days. If the link
 
 To activate your Sumo Logic Trial account:
 
-1. Check your inbox for the activation Sumo Logic email, then click **Activate Now**. If you don't see it, check your Spam and Trash folders.
-1. Fill out the **Activate Your Account** form.
-1. Click **Activate**.
-1. Once your account is activated, click the ![close](/img/get-started/github/close-icon.png) at the top right of the *Welcome to Sumo Logic* web page. You will not need to follow the in-application guide.
+1. In the Sumo Logic activation email, click **Activate account**. If you don't see it, check your Spam and Trash folders.
+1. Fill out the **Activate Your Account** form, then click **Activate**.
+
+Next, you'll be taken to a brief two-step data collector installation process, which will get you up and running with insights in a couple of minutes.
+
+### Install Data Collector
 
-In a few minutes, your account is activated and the Sumo Logic Setup Wizard displays in a new browser tab. You can **Collect, See, and Act** to monitor, troubleshoot, and secure your app.
+1. Select the platform (Linux, Windows, macOS, or AWS) where you'd like to install the Sumo Logic OpenTelemetry collector. <br/><img src={useBaseUrl('img/get-started/data-collection.png')} alt="icon" width="400" />
+1. Copy, paste, and run the provided command to your PowerShell or Terminal. This will install the collector, app, and dashboards. When finished, click **Start using Sumo**. <br/><img src={useBaseUrl('img/get-started/install-otel.png')} alt="icon" width="400" />
```

Comment:
```suggestion
1. Copy, paste, and run the provided command to your PowerShell or Terminal. This will install the collector, and relevent dashboards. When finished, click **Start using Sumo**. <br/><img src={useBaseUrl('img/get-started/install-otel.png')} alt="icon" width="400" />
```